app/mattermost.js

```diff
@@ -244,12 +245,21 @@ export default class Mattermost {
                 );
             } else if (state.entities.users && state.entities.users.currentUserId) {
                 setServerVersion(serverVersion)(dispatch, getState);
+
+                // Note that license and config changes are now emitted as websocket events, but
+                // we might be connected to an older server. Loading the configuration multiple
+                // times isn't a high overhead at present, so there's no harm in potentially
```

Comment:
I know this is true for server requests but is it true for re-renders as well?

Reply:
That's a good question. 

In practice, the subsequent configuration update should be identical to the one received over the websocket, but our reducer just replaces the existing configuration so redux will still reflect a change. React will do a render pass but nothing will have changed, but the render pass still takes some time.

Do we have any existing methodology for evaluating render performance?

Reply:
actually we don't have one, this shouldn't be that painful, but is not a bad idea to try and see if there are performance implications

Reply:
Honest question: how much effort do we want to spend on this? I'm imagining configuration changes occur only infrequently and evaluating and improving render performance in such an isolated event may not have any meaningful impact.

If we're really concerned about this, I could set a flag when a config/license websocket event occurs and skip the update when the server version changes thereafter. I initially deferred doing this as I anticipated we would eventually have a minimum server version past the websocket changes, but offhand I don't know how long that timeframe might be.

Reply:
I really don't think it would have a high impact on performance but I thought I should bring it up, I'm 0/5 what we do with this until the app catches up with the minimum  required server version

Reply:
Sounds good -- thanks for the feedback, @enahum. I'll update and submit this PR once https://github.com/mattermost/mattermost-redux/pull/418 is approved and I can update `yarn.lock`.

Reply:
Agreed that it will have little effect on performance, so it is fine this way. It'll only affect pre-4.8 servers when their config changes, so it'll be very rare